src/model/retrofit.jl

```diff
@@ -38,9 +37,32 @@ function make_retrofit_options(system::System, data::Dict{Symbol,Any})
 
     end
 
-end 
+end
+
+@doc raw"""
+    add_retrofit_constraints!(system::System, model::Model)
```

Comment:
This function signature is missing an argument (`period_idx`)
```suggestion
    add_retrofit_constraints!(system::System, period_idx::Int, model::Model)
```